common/src/main/java/org/keycloak/common/util/ServerCookie.java

```diff
@@ -32,6 +32,22 @@
     private static final String tspecials = ",; ";
     private static final String tspecials2 = "()<>@,;:\\\"/[]?={} \t";
 
+    public enum SameSiteAttributeValue {
+        NONE("None"), // we currently support only SameSite=None; this might change in the future
+        STRICT("Strict"),
```

Comment:
If we only support _NONE_ at the moment, I'd rather to make it a constant and not advertise LAX or STRICT in the code.

Reply:
@abstractj Done. (We actually had only the NONE value there at some point during the development. ;))